### DISCUSSION TITLE
nuxt generate return empty html

### DISCUSSION THREAD

**Discussion by code-black-wizard** [nuxt/nuxt#8878] [Questions (Nuxt 2)]
I have nuxt project and in my nuxt.config i wrote sss: true, target: 'static'. Also i have array of routes and i added them in generate.routes. But after i do yarn build and yarn generate nuxt returned me same and empty html with <title>Page not found</title> in head tag. If i write ssr: false, nuxt return me correct html, but me needed ssr.

**Comment by ghost** [comment]
`sss: true` ≠ `ssr: true`
